guide/dep-manip.md

```diff
@@ -68,24 +68,47 @@ with more to be added in the future. The lookup REST endpoint should follow:
 </tr>
 </table>
 
+##### REST Timeouts and retries
+
+In case of a 504 response from DA, by default the operations will be retried after a waiting period of 30 seconds. This value can be optionally configured with `-DrestRetryDuration=<...>`, expressed in seconds.
+
+The underlying HTTP client library responsible for calling the REST endpoints is set by default with a socket timeout of 10 minutes, and a connection timeout of 30 seconds. The values can be optionally configured respectively with `-DrestSocketTimeout=<...>` and `-DrestConnectionTimeout=<...>`, expressed in seconds.
+
+##### REST Headers
+
+You can add custom HTTP headers to your REST calls by setting a property named `restHeaders`. HTTP headers must be comma-separated and each non-empty name-value pair must be colon-separated.
+
+For example:
+
+    -DrestHeaders=log-user-id:102,log-request-context:061294ff-088,log-process-context:,log-expires:,log-tmp:
+
 ### Disabling Dependency Manipulation
 
 If the parameter `-DdependencySource=NONE` is set then this will disable communication with the REST source which effectively disables manipulation. Note that the version will still be changed - although, if there is no preceeding `manipulation.json` it will always get a `-00001` suffix.
```

Comment:
It looks like "preceeding" is misspelled, but I wasn't sure of the meaning in this case.

Reply:
I didn't alter that sentence! I'll fix the spelling.